code/modules/mob/status_procs.dm

```diff
@@ -20,13 +20,15 @@
 	dizziness = max(amount, 0)
 
 /////////////////////////////////// EYE_BLIND ////////////////////////////////////
-
 /**
   * Adjust a mobs blindness by an amount
   *
   * Will apply the blind alerts if needed
   */
-/mob/proc/adjust_blindness(amount)
+/mob/proc/adjust_blindness(amount, force)
+	//snowflake-ish fix for psyphoza
+	if(!force && HAS_TRAIT_FROM(src, TRAIT_BLIND, "uncurable"))
```

Comment:
this should just check if they even have eyes, which.. psyphoza don't? do they? If so it should be some var like `eyes_can_see`